compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPULowerExecutableTarget.cpp

```diff
@@ -104,6 +104,9 @@ void LLVMCPULowerExecutableTargetPass::runOnOperation() {
   pipelineOpts.enableUkernels = hasUkernel(target);
   pipelineOpts.enableAArch64SSVE =
       isAArch64(target) && hasAnySVEFeature(target) && hasSMEFeature(target);
+  pipelineOpts.enableAArch64Neon = isAArch64(target) && hasNeonFeature(target);
+  pipelineOpts.enableAArch64I8mm =
+      isAArch64(target) && hasI8mmFeature(target) && hasNeonFeature(target);
```

Comment:
I would also remove `hasNeonFeature` from 109. `+i8mm` includes `+neon` but we are not expanding all the features in IREE.